src/paymentsLoader.ts

```diff
@@ -40,12 +40,12 @@ export function RevolutPaymentsLoader(
         loadedPaymentInstance = RevolutCheckout.payments
         delete window.RevolutCheckout
       } else {
-        handleError()
+        handleError('RevolutCheckout is not a function')
       }
     }
 
     script.onload = handleLoad
-    script.onerror = handleError
+    script.onerror = () => handleError('Network error encountered')
```

Comment:
> Will onerror be called in case browser failed to parse the script due to e.g. an unsupported syntax? Then the error is not quite accurate

looks like it will not